_posts/2019-02-05-canvas-spheres.html

```diff
@@ -0,0 +1,112 @@
+---
+title: Canvas spheres
+thumbnail: -SITE_URL-/images/canvas-spheres.jpg
+excerpt: "An experiment with HTML5 canvas, requestAnimationFrame and ESLint."
+# githublink: "#"
+bitbucketlink: "https://bitbucket.org/Dries_Sercu/vanilla-js-canvas-spheres"
+demolink: "-SITE_URL-/projects/vanilla-canvas-spheres/index.html"
+---
+<p>
+	Inspired by a bed sheet from Ikea I wanted to make a small app that copied the look. You can use it as a screensaver too!
+	Having a visual result of the code I wrote was very rewarding.
+	And what should have been a small experiment ended up taking more of my time than I anticipated because I was enjoying myself.
+</p>
+
+{% include post-image.html image-link="-SITE_URL-/images/canvas-spheres-inspiration.jpg" image-description="The IKEA Smörboll bed sheets." %}
+
+<h2>Canvas</h2>
+
+<p>
+	Using the canvas element it is possible to draw and animate shapes using javascript.
+</p>
+<p>
+	So, what does it do?
+</p>
+<p>
+	The application will draw sphers (circles) in multiple rows.
```

Comment:
The application will draw spheres in multiple rows. (don't include circles between brackets, they smort)